build.gradle

```diff
@@ -317,19 +313,19 @@ dependencies {
     implementation("io.github.pixee:java-security-toolkit:1.2.1")
 
     // Exclude Tomcat and include Jetty
-    implementation("org.springframework.boot:spring-boot-starter-web:$springBootVersion")
```

Comment:
why remove this hardcoded verisons? is this to have plugin manage?

Reply:
with `id "io.spring.dependency-management" version "1.1.7"` is the dependency between the modules provided by https://spring.io/projects/spring-framework and by `id "org.springframework.boot" version "3.4.3"` the corresponding version is specified